build.cake

```diff
@@ -1,6 +1,6 @@
 #addin nuget:?package=Cake.Git&version=0.21.0
 
-var IOSVERSION = Argument("iosversion", "9.3.0");
+var IOSVERSION = Argument("iosversion", "9.3.2");
```

Comment:
I do not think this is correct since this should always match the current released iOS SDK

Reply:
Ah I see there is a new release, are there any API changes at all?

Reply:
Shouldn't be, just a patch release :) Thanks for checking this so quickly though!